### PR TITLE
Feature/logo as wagtail conf setting

### PR DESCRIPTION
**Pull request by codiebeulaine**
@miltontony @saeedmb8 ready for quick review

### CODE REVIEW

**Comment by codiebeulaine** [comment]
screen shot 2016-03-23 at 11 59 33 am

**Comment by smarzban** [comment]
:+1: :cake:

**Comment by miltontony** [comment]
holdup! Don't merge yet

File: freebasics/models.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from wagtail.contrib.settings.models import BaseSetting, register_setting
+from wagtail.wagtailimages.edit_handlers import ImageChooserPanel
+from django.db import models
+
+
+@register_setting
+class HomePageSettings(BaseSetting):
```
Comment (miltontony) [inline freebasics/models.py]: I think we should call this `SiteSettings` because we might need to add other settings here

Comment (miltontony) [inline freebasics/models.py]: You'll also need to generate migrations I think

**Comment by miltontony** [comment]
:cake: thanks for fixing the `{% trans %}` blocks.

File: freebasics/models.py (hunk 1 above)
Comment (codiebeulaine) [inline freebasics/models.py]: I tried doing migrations but it said no changes detected. Changing the name now

File: freebasics/templates/search/search_results.html [hunk 2]
```diff
@@ -14,39 +14,41 @@ <h6 class="sub-heading">{% trans "Search results for " %}{{search_query}}</h6>
...
+  </li>
+</ul>
+</div>
 
   {% else %}
       <div class="block-search fb-body fb-card">
-        <h6 class="block-heading fb-accent-1">Search Results</h6>
-        <h6 class="sub-heading">No search results for: {{search_query}}</h6>
+        <h6 class="block-heading fb-accent-1">{% trans "Search Results" %}</h6>
+        <h6 class="sub-heading">{% trans "Search results for: " %} {{search_query}}</h6>
```
Comment (miltontony) [inline freebasics/templates/search/search_results.html:47]: Hmm..can you put the colon outside the trans block please

File: freebasics/templates/search/search_results.html [hunk 3]
```diff
@@ -4,8 +4,8 @@
 {% block content %}
 {% if search_results %}
 <div class="block-search fb-body fb-card">
-  <h6 class="block-heading fb-accent-1">Search Results</h6>
-  <h6 class="sub-heading">{% trans "Search results for " %}{{search_query}}</h6>
+  <h6 class="block-heading fb-accent-1">{% trans "Search Results" %}</h6>
+  <h6 class="sub-heading">{% trans "Search results for: " %}{{search_query}}</h6>
```
Comment (miltontony) [inline freebasics/templates/search/search_results.html:8]: Same here - colon outside the trans block

**Comment by codiebeulaine** [comment]
slack plus one from @miltontony

**Pull request merged**